src/components/MobileNav.js

```diff
@@ -38,12 +70,16 @@ const CDPDropdown = ({ children }) => {
             justifyContent="center"
             px="m"
             py="s"
-            bg="teal.500"
+            bg="black.500"
             borderRadius="4px"
           >
-            <Text t="p6" fontWeight="bold">
-              ETH
-            </Text>
+            <NavbarIcon
+              prefetch={true}
```

Comment:
doesn't seem to be used?

Reply:
I was noticing the cdp data was "flickering" in and out, I thought using prefetch would solve that. Most likely the issue was due to using `useEffect` incorrectly as discussed below.